src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx

```diff
@@ -30,6 +30,8 @@ const BookingPill = styled.div<{ disabled: boolean; altStyle: boolean; color: st
     !disabled &&
     `
   cursor: pointer;
+  position: relative;
+  top:-8px;
```

Comment:
You're using absolute units to position this, but padding is in relative units.
Would reducing the `padding` fix this, rather than having to use a negative placement?

Reply:
Make sure that this looks okay at multiple zoom levels.
I recommend 75%, 100%, and 150%, and 200% at a minimum.

Reply:
Good spot, will change it